test/rouge_filter_test.rb

```diff
@@ -88,4 +88,13 @@ def test_lexer_can_be_specified
 
     assert_kind_of Rouge::Lexers::Shell, filter.lexer_for("shell")
   end
+
+  def test_replacing_br
+    filter = RougeFilter.new \
+      "<pre lang='ruby'>hello<br>world</pre>", replace_br: true
+
+    doc = filter.call
+    assert_equal "<pre class=\"highlight highlight-ruby\"><code>"\
+                 "<span class=\"n\">hello</span>\n<span class=\"n\">world</span></code></pre>\n", doc.to_html
```

Comment:
@JuanitoFatas I would suggest use 120 characters, since screen is so big these days...